specification/metrics/semantic_conventions/runtime-environment-metrics.md

```diff
@@ -297,6 +298,21 @@ This metric is obtained from [`ClassLoadingMXBean#getLoadedClassCount()`](https:
 <!-- semconv metric.process.runtime.jvm.classes.current_loaded(full) -->
 <!-- endsemconv -->
 
+### Metric: `process.runtime.jvm.available_processors`
```

Comment:
If there is the intent to add other JVM-specific processor details later on, we could introduce it as a namespace and call it `process.runtime.jvm.processors.available` instead.

Reply:
Maybe process.runtime.jvm.cpu.count, similar to the proposed process.runtime.jvm.cpu.time (#55)?

Reply:
hmm...I feel that `process.runtime.jvm.cpu.count` sounds like it could be the number of CPUs the process is using (which isn't the same as the number of CPUs available).

Reply:
given we already have:
- `process.runtime.jvm.cpu.time`
- `process.runtime.jvm.cpu.load`

would this metric fit with the name...?
- `process.runtime.jvm.cpu.available_processors`

Reply:
Maybe we should put this PR on hold. Ideally we would want to align with the name used under `process.cpu.*` for this same concept of "available processors", but there is no such semantic attribute yet (there was a previous proposal at https://github.com/open-telemetry/opentelemetry-specification/pull/2392).

Reply:
let's watch #99 (and comment if we have concerns about that name), it's proposing `system.cpu.count` which I think(?) would set precedent for us to use `process.runtime.jvm.cpu.count`

Reply:
#99 went with

* `system.cpu.logical.count`
* `system.cpu.physical.count`

since I think the JVM (and processes in general) don't have this distinction (and it's just "logical count"), I renamed the metric in this PR to

* `jvm.cpu.count`